packages/excalidraw/tests/__snapshots__/history.test.tsx.snap

```diff
@@ -12559,7 +12556,7 @@ exports[`history > singleplayer undo/redo > should create new history entry on i
 
 exports[`history > singleplayer undo/redo > should create new history entry on image drag&drop > [end of test] number of elements 1`] = `1`;
 
-exports[`history > singleplayer undo/redo > should create new history entry on image drag&drop > [end of test] number of renders 1`] = `7`;
+exports[`history > singleplayer undo/redo > should create new history entry on image drag&drop > [end of test] number of renders 1`] = `8`;
```

Comment:
Because, now, we perform `actionFinalize` after any image insert. Previously, `actionFinalize` was performed after inserting image via toolbar only